### PR TITLE
Release version 0.6.0

### PR DESCRIPTION
**Pull request by StanczakDominik**
What it says on the tin!

### CODE REVIEW

**Comment by StanczakDominik** [comment]
Will continue soon, brb, doggo!

**Comment by StanczakDominik** [comment]
All right, this should be good to go for a minor review. Definitely skip c1425e0. I think we could squash merge this one and see what happens; the tag should still be there, so I don't think anything will break.

Once that's merged, I'll try adding a v0.7.dev tag to master, so that the changelog entry works nicely. This will take an additional change to CI, which I'm adding in 31684cce.

File: docs/development/release_guide.rst [hunk 1]
```diff
@@ -73,7 +75,9 @@ Release
 
 * Make sure that tests pass and that documentation builds without issue.
 
-* No, really, check twice.
+  * No, really, check twice. Let the tests do their thing. You want things tip
+    top, and by now, you want that cuppa tea anyway. Treat yourself! Celebrate
+    the new release and let the darn tests pass.
```
Comment (rocco8773) [inline docs/development/release_guide.rst:80]: ☕️☺️

**Review by rocco8773 — APPROVED** [review]
Looks good to me! Thank you for all the hard work!

**Pull request merged**